middlewares/roles.js

```diff
@@ -0,0 +1,40 @@
+import { roles } from '../helpers/roles';
+import RoleAccess from '../helpers/accessServices';
+
+/**
+ * @author JABIRO Christian
+ * @class
+ * @classdesc Role
+ */
+class Role {
+  /**
+   *@function isUser
+   * @param {object} req
+   * @param {object} res
+   * @param {object} next()
+   * @returns {object} object
+   */
+  static async isAdmin(req, res, next) {
+    const findUser = await RoleAccess.findOne(req.user.id);
+    if (findUser.roles !== roles.isUser || findUser.roles !== roles.isSuperAdmin) {
+      return res.status(404).json({ status: 404, message: 'You are an admin!' });
```

Comment:
Does this response correspond to this status code? Change it, please.

Reply:
@ericnyirimana the status code is correct, though I need to correct the message.